Src/Sarif.PatternMatcher.Function/SpamAnalyzer.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+using Microsoft.CodeAnalysis.Sarif.Driver;
+using Microsoft.CodeAnalysis.Sarif.Writers;
+
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Function
+{
+    internal static class SpamAnalyzer
+    {
+        internal static readonly IFileSystem FileSystem;
+
+        static SpamAnalyzer()
+        {
+            FileSystem = Sarif.FileSystem.Instance;
+        }
+
+        public static SarifLog Analyze(string filePath, string text, string rulePath)
+        {
+            IEnumerable<string> regexDefinitions = FileSystem.DirectoryGetFiles(Path.Combine(rulePath, @"..\Rules"), "*.json");
```

Comment:
>..\Rules" [](start = 105, length = 9)

We shouldn't be inventing a new directory layout for this functionality, otherwise we can't install our plugins via nuget as part of deploying this function.

Eddy will understand what I'm talking about. It's important that we get this code deployed ASAP and then we can clean up all the things I mention. :)